app/src/main/java/io/intrepid/contest/rest/ContestStatusResponse.java

```diff
@@ -1,6 +1,37 @@
 package io.intrepid.contest.rest;
 
 public class ContestStatusResponse {
-    public boolean waitingForSubmissions;
-    public int numSubmissionsMissing;
+    private boolean submissionsEnded;
+    private int entriesSubmitted;
+    private int totalEntries;
+
+    private boolean contestEnded;
+    private int judgesSubmitted;
+    private int totalJudges;
+
+    public ContestStatusResponse(boolean submissionsEnded, boolean contestEnded,
```

Comment:
Yeah, I was debating whether or not to do it, but as in most cases all of those variables need to be set for the tests and mocks, I thought the constructor was the best place to enforce that.
However it was not obvious what the values were when using it; now I split it in two setters (one for submissions and one for judges)